packages/php-wasm/node/src/test/php-request-handler.spec.ts

```diff
@@ -102,32 +134,133 @@ describe.each(SupportedPHPVersions)(
 			});
 		});
 
-		it('should yield x-file-type=static when a static file is not found', async () => {
+		const fileNotFoundFallbackTestUris = [
+			'/index.php',
+			'/other.php',
+			'/index.html',
+			'/testing.html',
+			'/',
+			'/subdir',
+		];
+		fileNotFoundFallbackTestUris.forEach((nonExtistentFileUri) => {
+			it(`should relay a fallback response for non-existent file: '${nonExtistentFileUri}'`, async () => {
+				getFileNotFoundActionForTest = (uri: string) => {
+					if (uri === nonExtistentFileUri) {
+						return {
+							type: 'response',
+							response: new PHPResponse(
+								404,
+								{ 'x-file-type': ['static'] },
```

Comment:
X-file-type static doesnt mean much anymore in this context when it also works for .php paths, it would be nice to tell the service worker „please fetch from the server” using a different object shape

Reply:
(I’m just pointing at the string used)

Reply:
Yeah, it's not great. Unfortunately, I haven't figured out how to adjust this and not break for clients still running with the older Service Worker:

From [this comment](https://github.com/WordPress/wordpress-playground/pull/1539#discussion_r1655810700):

> > That's a good opportunity to rename this to more clearly communicate the intention. Perhaps 'x-file-source': 'remote-server', 'x-backfill-from': 'remote-host', something similar?
>
> @adamziel Is it OK for us to leave this magic header as-is for now? I made the update but quickly discovered that running with an older Service Worker causes remote asset requests to fail until the updated Service Worker becomes active.

One thing we could do for now is send the new preferred header while continuing to send the old, like:

```js
{
    'x-backfill-from': [ 'remote-host' ],
    // Include this header for remote assets via out-of-date service worker
    'x-file-type': [ 'static' ],
}
```

I'm planning to go this route for now.